ink-wrapper-types/src/lib.rs

```diff
@@ -1,57 +1,16 @@
-#[cfg(feature = "aleph_client")]
-mod aleph_client;
+#[cfg(feature = "drink")]
+pub use crate::drink::*;
+#[cfg(feature = "drink")]
+mod drink;
```

Comment:
I'd like to leave the two, even though they may not be necessary right now, so that in the future the alternative backends are easy to add. I had spent considerable amount of time on this and corresponding `Cargo.toml` juggling to figure this out.

Reply:
:+1: